ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputShadowNode.java

```diff
@@ -249,4 +249,8 @@ public void setPadding(int spacingType, float padding) {
     super.setPadding(spacingType, padding);
     markUpdated();
   }
+
+  protected EditText createDummyEditText() {
```

Comment:
Let's call this something like `createInternalEditText` or `createEditTextForMeasure` since this is becoming a public API.

Reply:
Let's also add a javadoc here to document what this is for and how it can be used.

Reply:
👍  Makes sense. I also updated the field from `mDummyEditText` to `mInternalEditText` for consistency.